pom.xml

```diff
@@ -3334,7 +3334,7 @@
     -->
```

Comment:
there are a few more places mentioned `Hadoop-3.2`, better to replace them too.

Reply:
Replace to `Hadoop-3.x` is OK?